src/include/sof/audio/chain_dma.h

```diff
@@ -0,0 +1,41 @@
+#include <sof/audio/buffer.h>
+#include <sof/audio/component.h>
+#include <sof/audio/component_ext.h>
+#include <sof/audio/pipeline.h>
+#include <sof/common.h>
+#include <sof/ipc/topology.h>
+#include <sof/ipc/common.h>
+#include <ipc/dai.h>
+
+static const uint32_t MAX_CHAIN_NUMBER = DAI_NUM_HDA_OUT + DAI_NUM_HDA_IN;
+
+/* chain dma component private data */
+struct chain_dma_data {
+	bool first_data_received_;
```

Comment:
why are you using underscore at the end? Don't think this is a common practice in SOF